tests/expected/function-contract/history/simple_pass_twice.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// kani-flags: -Zfunction-contracts
+
+#[kani::ensures(|result| old(*ptr + 1) == *ptr)]
+#[kani::ensures(|result| old(*ptr + 1) == *ptr)]
+#[kani::requires(*ptr < 100)]
+#[kani::modifies(ptr)]
+fn modify(ptr: &mut u32) {
+    *ptr += 1;
+}
+
+#[kani::proof_for_contract(modify)]
```

Comment:
What about test cases that use `stub_verified`? These are important because `old` can help us to write more precise ensure clauses, which would restrict more the value havoced during replacement. Could you add some of those as well? Also, what happen if I use old without an modifies clause? Would this lead to an error?